styx-scheduler-service/src/test/java/com/spotify/styx/flyte/FlyteAdminClientRunnerTest.java

```diff
@@ -125,14 +133,32 @@ public void testCreateExecution() throws FlyteRunner.CreateExecutionException {
         .build();
   }
 
+  @Test
+  @Parameters(method = "styxTriggerToFlyteExecMode")
+  public void testStyxTriggerTranslateToFlyteExecutionMode(Trigger styxTrigger, ExecutionMode flyteExecMode)
+      throws FlyteRunner.CreateExecutionException {
+    flyteRunner.createExecution(runState(styxTrigger), "test-create-execution", FLYTE_EXEC_CONF);
```

Comment:
Do we also want to test what happens when there isn't a Trigger?  (not very important)

Reply:
I added `new Object[] { null, ExecutionMode.UNRECOGNIZED }` for testing no trigger